Documentation/rhel-machine-setup-and-ci.md

```diff
@@ -18,6 +18,8 @@
 ## CI Machines
 
 - Our RHEL CI machines are dnsb-rhel7-1, dnsb-rhel7-2, dnsb-rhel8-1, and dnsb-rhel8-2 in the DotNet-SourceBuild-Internal pool.
```

Comment:
I'm not certain any of this should be in the open.  It is not disclosing anything but there it is not intended for OSS contributiors.  It should all be in our OneNote IMO.  We want to avoid accidental disclosure.